ext/RMagick/extconf.rb

```diff
@@ -3,7 +3,12 @@
 
 module RMagick
   class Extconf
-    require 'rmagick/version'
+    if RUBY_VERSION < '1.9.2'
+      require File.expand_path('../../lib/rmagick/version',
+                               File.dirname(__FILE__))
+    else
+      require_relative '../../lib/rmagick/version'
+    end
```

Comment:
Yeah, load path, That's true!
I'll fix it. Thank you.

> If these both work, then you only need the 1.9.2 form.

FWIW, I thought so, but something might be different...

fa98023 #142
```
require_relative '../../lib/rmagick/version'
```

8cf3380
```
require File.expand_path('../../../../lib/rmagick/version')
```

6b24554
```
require File.expand_path('../../lib/rmagick/version')
```